src/Codenizer.HttpClient.Testable/RequestBuilder.cs

```diff
@@ -396,6 +408,11 @@ internal Dictionary<string, string> BuildRequestHeaders()
         {
             var headers = new Dictionary<string, string>();
 
+            if (AuthorizationHeader is not null)
+            {
+                headers.Add("Authorization", AuthorizationHeader.ToString());
```

Comment:
I was a bit worried that this could lead to an empty string but I've verified that calling `new AuthenticationHeaderValue("", "")` throws so this is actually safe.